packages/amplify-graphql-schema-generator/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aws-amplify/graphql-schema-generator",
-  "version": "0.1.8",
+  "version": "0.2.0",
```

Comment:
Yeah, this is changing, and I don't think should be (this is probably triggering that other change in the root `package.json` file

Reply:
I ran into an issue while publishing a tagged release with 0.1.8 so I manually upgraded to 0.2.0 and it worked. So keeping it as it is.